client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx

```diff
@@ -6,14 +6,20 @@ import useStyles from './commentDisplayStyles';
 
 const noCommentMessage = 'אין הערה';
 
-const CommentDisplay = ({comment}: Props) => {
+const CommentDisplay = ({comment, scrollableRef}: Props) => {
     const [isTooltipOpen, setIsTooltipOpen] = React.useState<boolean>(false);
+
     const handleTooltipClose = () => setIsTooltipOpen(false);
     const handleTooltipToggle = (event:any) => {
         event.stopPropagation();
         setIsTooltipOpen(isOpen => !isOpen);
     };
 
+    React.useEffect(() => {
+        scrollableRef?.addEventListener('scroll', handleTooltipClose)
+        return () => scrollableRef?.removeEventListener('scroll', handleTooltipClose)
+    });
```

Comment:
I think you should add a useEffect condition only when the ref changes.

Reply:
If i understand this correctly a new event listener will be created on each render

Reply:
👍  yup definitely.